### DISCUSSION TITLE
What does the wakatime card do?

### DISCUSSION THREAD

**Discussion by SnepalDelaupe** [anuraghazra/github-readme-stats#2291] [Q&A] [answered]
What does the wakatime card do?

**Comment by MitkoVtori** [comment, marked as answer]
It's a weekly card, It tells if you have activity the current week.

**Reply by rickstaa** [reply]
@MitkoVtori Thanks for helping us out I saw your collaboration message some days ago but I couldn't find it anymore when I wanted to reply. But welcome to the community 🚀.

**Reply by rickstaa** [reply]
For more information about the stats on the WakaTime card, see https://wakatime.com.
